packages/actor-init-sparql/test/MediaTypeSelection-test.ts

```diff
@@ -0,0 +1,105 @@
+import minimist = require('minimist');
```

Comment:
Creating a separate test file doesn't feel like a good solution here. The convention is that each class has a corresponding test file. But `MediaTypeSelection` is not an existing class, so we shouldn't have this here.

Instead, I would just create a new describe block (`'determineMediaType'`) under the `'HttpServiceSparqlEndpoint'` describe block.
If you run into problems with conflicting mocks, I think you can scope them to specific describe blocks, but I'm sure the jest documentation can tell you more about that.